src/server.js

```diff
@@ -0,0 +1,31 @@
+require('require-self-ref')
+
+require('~/src/config').load()
+
+const startupTasks = require('~/src/startup-tasks')
+
+;(async function () {
+  try {
+    await startupTasks.startAll()
+
+    const logger = require('~/src/logging').logger(module)
+    const config = require('~/src/config')
+
+    logger.info(`Started all tasks in environment ${config.getEnvironment().name()}`)
+  } catch (err) {
+    console.log('Failed to start server: ', err)
+    return process.exit(1)
+  }
+})()
+
+process.on('uncaughtException', function (err) {
+  // TODO: Log this to a database
+  console.error('Uncaught exception: ', err)
+  process.exit(1)
+})
+
+process.on('unhandledException', function (err) {
+  // TODO: Log this to a database
```

Comment:
For situations like this, there's not real guarantee that we will have access to a our database at this point. Using a monitor or logging system like newrelic or scaylr would be slightly more reliable.